api/api/versions_api.go

```diff
@@ -79,13 +80,24 @@ func (c *VersionsController) PatchVersion(r *http.Request, vars map[string]strin
 func (c *VersionsController) ListVersions(r *http.Request, vars map[string]string, _ interface{}) *Response {
 	ctx := r.Context()
 
+	var query service.VersionQuery
+	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
```

Comment:
What will happen if `.Query()` is empty?

Reply:
it will use default value of VersionQuery struct